addons/account/models/res_partner_bank.py

```diff
@@ -16,14 +16,34 @@ class ResPartnerBank(models.Model):
     journal_id = fields.One2many(
         'account.journal', 'bank_account_id', domain=[('type', '=', 'bank')], string='Account Journal', readonly=True,
         help="The accounting journal corresponding to this bank account.")
+    display_iban_warning = fields.Boolean(
+        compute='_compute_display_account_warning',
+        help='Technical field used to display a warning if the IBAN country is different than the holder country.',
+        store=True,
+    )
+    display_wise_warning = fields.Boolean(
```

Comment:
I would not name this field `wise` and make it generic. WISE is certainly not the only low-quality/scammer-friendly institution.